Author: Ilya Barygin <randomaction@ubuntu.com>
Description: - Add AM_PROG_CC_C_O required by newer automake versions.
- Avoid autoconf errors related to AM_PATH_GLIB, AM_PATH_GTK, and AM_PATH_XMMS
  by using pkg-config instead.
- audiofile >= 0.3.2-1 does no longer provide audiofile-config and the
  AM_PATH_AUDIOFILE macro based on it. Use pkg-config instead (see #655181).
--- a/configure.ac
+++ b/configure.ac
@@ -12,6 +12,7 @@
 
 AC_CANONICAL_HOST
 AC_PROG_CC
+AM_PROG_CC_C_O
 AC_ISC_POSIX
 AC_PROG_INSTALL
 AC_PROG_LN_S
@@ -221,9 +222,9 @@
 
 dnl *** Stuff for xmms plugin ***
 have_xmms=true
-AM_PATH_GLIB(1.2.2, , [ have_xmms=false ])
-AM_PATH_GTK(1.2.2, , [ have_xmms=false ], gthread)
-AM_PATH_XMMS(1.0.0, , [ have_xmms=false ])
+PKG_CHECK_MODULES([GLIB], [glib-2.0], , [ have_xmms=false ])
+PKG_CHECK_MODULES([GTK], [gtk-2.0], , [ have_xmms=false ])
+PKG_CHECK_MODULES([XMMS], [xmms2-client], , [ have_xmms=false ])
 AM_DISABLE_STATIC
 AM_PROG_LIBTOOL
 AC_ARG_ENABLE(xmms,
@@ -249,7 +250,8 @@
 
 dnl *** Stuff for audiofile library ***
 dnl v0.2.1 and before have bugs with 24-bit LE files
-AM_PATH_AUDIOFILE([ 0.2.2 ], [ have_audiofile=true ])
+PKG_CHECK_MODULES([AUDIOFILE], [audiofile >= 0.2.2], [have_audiofile=true], [])
+dnl AM_PATH_AUDIOFILE([ 0.2.2 ], [ have_audiofile=true ])
 dnl AC_CHECK_LIB(audiofile, afSetVirtualSampleFormat, have_audiofile=true)
 AH_TEMPLATE([USE_AUDIOFILE],
 	    [Define if you want to use the audiofile library.])
